### ISSUE TITLE
enable event_cpi feature for anchor_lang

### ISSUE THREAD

**Issue by IceCY** [solana-playground/solana-playground#255]
Hi, it would be useful if anchor_lang's `event_cpi` feature is enabled.  It is enable with `anchor-lang = { version = "0.xx.x", features = ["event-cpi"] }`. Would you like to enable it ?
Thanks.

**Comment by acheroncrypto** [comment]
Yeah, this is quite straightforward to add. However, there is an unrelated problem: the server we host currently has a problem and needs to be migrated before we can deploy the changes.
